src/components/forms/sections/UnmarkedWinterOverview.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Col, Container, Row } from 'reactstrap';
+import { UnmarkedWinterFormValues, WinterStorageArea } from '../../../types/unmarkedWinterStorage';
+
+import Agreement from '../fragments/Agreement';
+import Newsletter from '../fragments/Newsletter';
+import UnmarkedWinterOverviewInfo from './UnmarkedWinterOverviewInfo';
+
+import { StepType } from '../../steps/step/Step';
+import { WithBoatType } from '../Selects';
+
+type Props = {
+  values?: UnmarkedWinterFormValues;
+  selectedArea: WinterStorageArea;
+  boatTab: string;
+  steps: StepType[];
+} & WithBoatType;
+
+const UnmarkedWinterOverview = ({ values, selectedArea, boatTab, boatTypes, steps }: Props) => {
```

Comment:
these headers deviate from Abstract, they are defined in Abstract as "Talvisäilytysalue" and "Omistajan tiedot":
<img width="720" alt="Screenshot 2020-08-31 at 14 30 27" src="https://user-images.githubusercontent.com/1407958/91715930-5234c300-eb97-11ea-8d51-e6705a7fa016.png">

Reply:
Good catch, thanks! I had this in mind, but looks like I forgot it.